Yoast/wordpress-seo
# Floating Save Changes button for admin pages.

## Summary


This PR can be summarized in the following changelog entry:

* Show a floating `Save changes` button on Yoast SEO admin pages when the normal button is not visible in the browser window.

## Relevant technical choices:

* Added a "private" jQuery utility function to determine whether an elment is visible in the browser's viewport.

## Test instructions

This PR can be tested by following these steps:

* Test different window sizes on the SEO settings pages an make sure a save button is always visible.

## UI changes

## Documentation

## Quality assurance


Fixes #12626
